vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -621,7 +622,8 @@ base_learner* ccb_explore_adf_setup(options_i& options, vw& all)
                .necessary()
                .help(
                    "EXPERIMENTAL: Do Conditional Contextual Bandit learning with multiline action dependent features."))
-      .add(make_option("all_slots_loss", all_slots_loss_report).help("Report average loss from all slots"));
+      .add(make_option("all_slots_loss", all_slots_loss_report).help("Report average loss from all slots"))
+      .add(make_option("ccb_model_without_has_seen_flag", data->input_model_without_has_seen_multi_slot_flag).help("Legacy ccb model format without HasSeenMultipleSlots flag"));
```

Comment:
does this have to be exposed to the end user? Could we do the `is_ccb_run` and `is_ccb_model` checks in conditional_contextual_bandit setup and set the boolean there?

Reply:
we are merging options from command line and model before entering stack of setups, so when we first time entersconditional_contextual_bandit::setup information about which options came from model and which are from command line is already lost (and there is no way to populate is_ccb_run and is_ccb_model here).

Reply:
I see, good point... would something like `ccb_with_cb_model` work?

Reply:
This option could be hidden too

Reply:
what does it mean?

Reply:
@lalo I thought we added hidden? I can't seem to find it though

Reply:
I deleted it, but the impl is pretty brief see commit: https://github.com/VowpalWabbit/vowpal_wabbit/commit/4ee50a8ddbea104013f9d8540b6ea2902bb9acfe

options.h and options_boost_po.cc

Reply:
*although it was for options groups and not individual options